app/src/androidTest/java/chat/rocket/android/authentication/loginoptions/ui/LoginOptionsFragmentTest.kt

```diff
@@ -0,0 +1,97 @@
+package chat.rocket.android.authentication.loginoptions.ui
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.action.ViewActions.scrollTo
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import chat.rocket.android.R
+import chat.rocket.android.authentication.ui.AuthenticationActivity
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@LargeTest
+class LoginOptionsFragmentTest {
```

Comment:
@GOVINDDIXIT This test should fail against our dev.veranda.viasat.io server because we have not configured login with facebook, google, linkdin etc.... Can you explain to me why it's passing, and testing against open.rocket.chat, even though I've set our SERVER = dev.veranda.viasat.io?

Reply:
@ear-dev this test is passing as it is testing the default login option fragment which is opened when we click join in the community button(open.rocket.chat) on onboarding fragment. The default loginOptionfragment doesn't ask for server it always open with (open.rocket.chat) as base URL. That's why it is passing.

Reply:
ok.  gottit, thanks.